### PR TITLE
resolved issue 2 and 3

### PR DESCRIPTION
**Pull request by basu0001**
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

### CODE REVIEW

File: dvc/analytics.py [hunk 1]
```diff
@@ -2,13 +2,18 @@
 
 from __future__ import unicode_literals
 
-from dvc.utils.compat import str
-
 import os
+
```
Comment (efiop) [inline dvc/analytics.py]: You shouldn't separate individual imports, but rather groups of imports. So this one should be:
```
from __future__ import unicode_literals

import os
import json
import errno
import logging

from dvc.utils.compat import str
from dvc import __version__

```

File: dvc/config.py [hunk 2]
```diff
@@ -2,16 +2,20 @@
 
 from __future__ import unicode_literals
 
-from dvc.utils.compat import str, open
-
 import os
```
Comment (efiop) [inline dvc/config.py]: Same here.

File: dvc/analytics.py (hunk 1 above)
Comment (basu0001) [reply dvc/analytics.py]: Done accordingly to this.

File: dvc/config.py (hunk 2 above)
Comment (basu0001) [reply dvc/config.py]: Done accordingly to this.

**Comment by basu0001** [comment]
@efiop pre-commit hooks installed any more changes required?

**Comment by efiop** [comment]
@basu0001 But the patch is still not formatted by the hook, I guess you need to run it manually now by running `pre-commit run --all-files`.

**Comment by efiop** [comment]
Hi @basu0001 ! Need any help with this? :slightly_smiling_face:

**Comment by efiop** [comment]
Closing due to inactivity. Please feel free to reopen the PR. 🙂Thanks!

**Pull request closed without merging**